src/Controls/samples/Controls.Sample.Sandbox/MainPage.xaml

```diff
@@ -3,5 +3,15 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
     x:Class="Maui.Controls.Sample.MainPage"
     xmlns:local="clr-namespace:Maui.Controls.Sample">
+  <VerticalStackLayout>
 
+    <Button Text="Add new window" Clicked="NewWindowButton_Clicked"/>
```

Comment:
I know this is testing, but we have a multi window test page in the sample already. Maybe add a button there.